packages/dds/tree/src/feature-libraries/chunked-forest/codec/compressedEncode.ts

```diff
@@ -262,18 +262,18 @@ export const anyFieldEncoder: FieldEncoder = {
 /**
  * Encodes a chunk using {@link EncodedInlineArray}.
```

Comment:
Maybe something like this would be helpful?

```suggestion
 * Encodes a chunk using {@link EncodedInlineArray}.
 * @remarks
 * The fact this is also a Shape is an implementation detail of the encoder: that allows the shape it uses to be itself, which is an easy way to keep all the related code together without extra objects.
```